aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -162,6 +162,10 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
         }
     }
 
+    private val REGEXP_EMAIL = Regex("^[A-Z0-9._%+-]+@[A-Z0-9.-]+.[A-Z]{2,4}$",
```

Comment:
Top level domains can be longer than 4 characters now (e.g. `.design`)

Reply:
Good point @koke . Any recommendation for how long to change it to? Some cursory googling has produced info for TLDs that can be evn 24chars long. Localized TLDs in particular start with `XN--` which is 4char long to begin with!

I think I will change it to 20chars.

Reply:
It seems that there is no set length limit for TLD. Checking on google I found think we can change it to 
`"^[A-Z0-9._%+-]+@[A-Z0-9.-]+.[A-Z]{2,}$"`

Reply:
Thanks @daniloercoli , lifted the upper limit with 3e9f744.